pkg/ccl/backupccl/restore_job.go

```diff
@@ -541,17 +541,18 @@ func restore(
 			return errors.Wrap(err, "sending remote AddSSTable requests")
 		}
 		md := restoreJobMetadata{
-			jobID:              job.ID(),
-			dataToRestore:      dataToRestore,
-			restoreTime:        endTime,
-			encryption:         encryption,
-			kmsEnv:             kmsEnv,
-			uris:               details.URIs,
-			backupLocalityInfo: backupLocalityInfo,
-			spanFilter:         filter,
-			numImportSpans:     numImportSpans,
-			execLocality:       details.ExecutionLocality,
-			exclusiveEndKeys:   fsc.isExclusive(),
+			jobID:                  job.ID(),
+			dataToRestore:          dataToRestore,
+			restoreTime:            endTime,
+			encryption:             encryption,
+			kmsEnv:                 kmsEnv,
+			uris:                   details.URIs,
+			backupLocalityInfo:     backupLocalityInfo,
+			spanFilter:             filter,
+			numImportSpans:         numImportSpans,
+			execLocality:           details.ExecutionLocality,
+			exclusiveEndKeys:       fsc.isExclusive(),
+			creationClusterVersion: job.Payload().CreationClusterVersion,
```

Comment:
I wonder if we really need to use the creation cluster version here. One all nodes are on the same version, it is safe to send the new message even if we were started on a lower version.  But, it's fine I suppose, most restores don't span versions and this makes us safe.

Reply:
Good point. Updated this to be the `ResumeClusterVersion`